.gitignore

```diff
@@ -45,28 +45,8 @@ __pycache__/
 *.py[cod]
 *$py.class
 
-# C extensions
-*.so
-
 # Distribution / packaging
-.Python
-build/
-develop-eggs/
 dist/
```

Comment:
personally, i'd just remove everything from gitignore we don't explicitly need. in this case, `lib` would ignore part of the resolver test case (and nobody should be writing eggs anymore anyway)

Also good news, venvs created by `python -m venv` will gitignore themselves from python 3.13 on: https://github.com/python/cpython/pull/108125. venvs created by `virtualenv` already gitignore themselves for a while already (https://github.com/pypa/virtualenv/pull/1825)

Reply:
I personally find this useful because it matches the exact GitHub-recommended `.gitignore`, so it's easy to maintain and update in the future if needed.